OrbitClientGgp/ClientGgp.cpp

```diff
@@ -164,6 +180,43 @@ bool ClientGgp::InitCapture() {
   return true;
 }
 
+void ClientGgp::InformUsedSelectedCaptureFunctions() {
+  if (capture_functions_used_.size() != options_.capture_functions.size()) {
+    for (std::string selected_function : options_.capture_functions) {
+      if (!capture_functions_used_.contains(selected_function)) {
+        ERROR("Function matching %s not found; will not be hooked in the capture",
+              selected_function);
+      }
+    }
+  } else {
+    LOG("All functions provided had at least a match");
+  }
+}
+
+bool ClientGgp::IsSelectedFunction(const FunctionInfo& func) {
+  for (std::string selected_function : options_.capture_functions) {
```

Comment:
Might profit from making it a const ref.

Reply:
True, will update the code, thanks!

Reply:
Done! I also missed the const in other loops so I updated them as well